packages/opentelemetry-exporter-prometheus/package.json

```diff
@@ -42,16 +42,20 @@
     "@types/node": "^12.6.9",
     "codecov": "^3.5.0",
     "gts": "^1.1.0",
-    "mocha": "^6.2.0",
+    "mocha": "^6.2.2",
     "nyc": "^14.1.1",
     "rimraf": "^3.0.0",
-    "tslint-microsoft-contrib": "^6.2.0",
     "ts-mocha": "^6.0.0",
     "ts-node": "^8.3.0",
+    "tslint-microsoft-contrib": "^6.2.0",
     "typescript": "^3.6.3"
   },
   "dependencies": {
+    "@opentelemetry/base": "^0.1.1",
     "@opentelemetry/core": "^0.1.1",
-    "@opentelemetry/types": "^0.1.1"
+    "@opentelemetry/metrics": "^0.1.1",
+    "@opentelemetry/types": "^0.1.1",
+    "prom-client": "^11.5.3",
+    "tslint-consistent-codestyle": "^1.16.0"
   }
 }
```

Comment:
Do you want to remove `"private": true` ?

Reply:
yes. didn't realize khan had it there